### PR TITLE
Migrated code from examples repo

### PR DESCRIPTION
**Pull request by jakemulf**

### CODE REVIEW

**Review by hythloda — APPROVED** [review]
works great! super easy set up and clear.  Only suggestion I have is change the start.sh to something more descriptive. Perhaps promDHstart.sh. I just know we will start getting a lot of start.sh.. but this is an organization/style chioce

File: README.md [hunk 1]
```diff
@@ -1 +1,40 @@
...
+## How it works
+
+This app runs a [Python Flask server](https://flask.palletsprojects.com/en/2.0.x/) wherever it is deployed.
+
+### Components
+
+* `alertmanager/config.yml` - The Prometheus alerts rules. This includes the configuration for the webhook alerts destination URL.
+* `docker-compose.yml` - Docker compose file that defines the Deephaven, Prometheus, and Python docker images.
+* `flask-app/requirements.txt` - Python dependencies for the application.
+* `flask-app/server.py` - The python server that accepts the Prometheus alert webhooks.
```
Comment (margaretkennedy) [inline README.md]:
```suggestion
* `flask-app/server.py` - The Python server that accepts the Prometheus alert webhooks.
```

File: README.md [hunk 2]
```diff
@@ -1 +1,40 @@
...
+* `flask-app/Dockerfile` - The Dockerfile for the python server.
```
Comment (margaretkennedy) [inline README.md]:
```suggestion
* `flask-app/Dockerfile` - The Dockerfile for the Python server.
```

**Comment by jakemulf** [comment]
> works great! super easy set up and clear. Only suggestion I have is change the start.sh to something more descriptive. Perhaps promDHstart.sh. I just know we will start getting a lot of start.sh.. but this is an organization/style chioce

Sure! We can try this out, I'll rename it to `promDhAlertsStart.sh` since there will be a few prometheus/deephaven apps

**Pull request merged**